### PR TITLE
Resolve issue so subPath: true works correctly

### PR DESCRIPTION
**Pull request by tnederlof**
This PR sets `mountPath` to `license.file.mountPath`/`license.file.secretKey` when `mountSubPath: true`. This is the correct syntax when using `subPath`.

Closes: https://github.com/rstudio/helm/issues/427

Also bumps each product to use the new `rstudio-library`

### CODE REVIEW

**Comment by tnederlof** [comment]
@colearendt @atheriel my first time PR here so let me know if something else is needed. I intend to add some related license file/secret documentation to Workbench, Connect and PM next so I figured I would bump `rstudio-library` version in those charts in that PR along with the doc changes and NEWS update next.

File: charts/rstudio-library/templates/_license-mount.tpl [hunk 1]
```diff
@@ -6,9 +6,14 @@
...
+  mountPath: "{{ .license.file.mountPath }}/{{ .license.file.secretKey }}"
   subPath: {{ .license.file.secretKey | quote }}
+  {{ else }}
+  mountPath: {{ .license.file.mountPath | quote }}
   {{- end }}
 {{- end }}
 {{- end -}}{{- /* end define template */ -}}
+
+
+
```
Comment (atheriel) [inline charts/rstudio-library/templates/_license-mount.tpl:19]: Nit: extra whitespace.

**Review by atheriel — APPROVED** [review]
This LGTM but should probably be tested manually if possible.

**Comment by tnederlof** [comment]
> This LGTM but should probably be tested manually if possible.

I was able to manually test it with Workbench, mounted correctly (was broken before). Also checked the pod YAML to see the mount path which matched what I expected.

**Comment by tnederlof** [comment]
@atheriel ended up bumping the products here

**Review by atheriel — APPROVED** [review]

File: charts/rstudio-library/templates/_license-mount.tpl [hunk 2]
```diff
@@ -6,9 +6,11 @@
...
-  mountPath: {{ .license.file.mountPath | quote }}
   {{- if .license.file.mountSubPath }}
+  mountPath: "{{ .license.file.mountPath }}/{{ .license.file.secretKey }}"
   subPath: {{ .license.file.secretKey | quote }}
+  {{ else }}
+  mountPath: {{ .license.file.mountPath | quote }}
   {{- end }}
 {{- end }}
-{{- end -}}{{- /* end define template */ -}}
+{{- end -}}{{- /* end define template */ -}}
```
[the remark this answers is not in the document]
Comment (tnederlof) [reply charts/rstudio-library/templates/_license-mount.tpl]: I appreciate it, better to get it right, never turned on after getting new comp!

**Pull request merged**